meetings/2019/2019-01-07.md

```diff
@@ -30,7 +30,7 @@ Extracted from **bootstrap-agenda** labelled issues and pull requests from the *
 * CPC Charter
   - 5 minute timebox
   - Refs:
-    - [Proposal Readme](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/proposals/approved/CPC_CHARTER/README.md)
+    - [Proposal Readme](https://github.com/nodejs/bootstrap/blob/03ba5897e9c4e01ac2bf6b118a9a5ea2b5b85f61/proposals/mdawson-cpc-charter/README.md)
```

Comment:
shouldn’t this be https://github.com/openjs-foundation/cross-project-council/blob/a376c1dabb01fb1f0ac66affa9104876dee76fa9/proposals/approved/CPC_CHARTER/README.md ?

If the concern is archival it seems like we’d want to avoid relying on repos outside of this org

Reply:
Agreed. Seems strange that it links out.

Reply:
My main concern is being able to cleanup the repo without making it really difficult to follow relevant links. I felt like the most reasonable decision was to pick the permalinks that people would have chosen back then. I do get your point however, although I feel like it's partially mitigated by Node being a foundation project.

Reply:
This is a long-winded way of saying: you're right but I hate having to do that work. :D

Reply:
go to the original link, and hit the key shortcut for copying the permalink? (it’s in the drop down menu button on the upper right). You can also hit y and copy the resulting URL.